third_party/terraform/utils/config.go.erb

```diff
@@ -268,7 +268,7 @@ func (c *Config) LoadAndValidate() error {
 	c.clientContainerBeta.UserAgent = userAgent
 	c.clientContainerBeta.BasePath = containerBetaClientBasePath
 
-	dnsClientBasePath := removeBasePathVersion(c.DnsBasePath) + "v1/projects/"
+	dnsClientBasePath := removeBasePathVersion(c.DNSBasePath) + "v1/projects/"
```

Comment:
Is there no TPUBasePath?

Reply:
This part only exists for products where we have both generated + handwritten resources.